website/project/views/node.py

```diff
@@ -856,7 +856,7 @@ def private_link_table(node, **kwargs):
 @must_be_valid_project
 def get_editable_children(auth, node, **kwargs):
 
-    if not node.can_edit(auth):
+    if not node.has_permission(auth.user, 'admin'):
```

Comment:
It looks like `get_editable_children` is used for both adding contributors AND for creating view-only links. While this change is correct for the former, it will prevent users from creating VOLs for projects they have write access to, which I believe is incorrect.